internal/util/porch/function.go

```diff
@@ -30,14 +29,14 @@ import (
 
 const expirationNanoSecond = 10
 
-var PorchExpiration = time.Duration(expirationNanoSecond) * time.Second
+var Expiration = time.Duration(expirationNanoSecond) * time.Second
```

Comment:
this feels a bit strange. A simpler way may be:

const Expiration time.Duration = 10 * time.Second

(I think Duration can be const)
the line 30 is actually not a duration, just an int so I would drop it as it doesn't seem required to the task.

Reply:
Thanks!